### ISSUE TITLE
Mobile Pop Up Dictionary not working 

### ISSUE THREAD

**Issue by bguati** [AnKing-VIP/ankiweb-placeholder#122]
## Which add-on is this about?
Mobile Pop Up Dictionary

#### Problem description

*Downloaded the add-on for desktop and set it up properly, but when I go on mobile devices and highlight a word, the popup dictionary doesn't show. *

![image](https://github.com/user-attachments/assets/6128e037-fe5f-40c3-b5c2-d43c833f1d8d)

![image](https://github.com/user-attachments/assets/4cf8bde5-52b1-45bc-ae9a-4ac5282b3a78)

I tried changing language, chaining card types etc, but didn't work.

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [x] I've restarted Anki to see if it helps
 - [x] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [ ] I've tried to disable other add-ons to see if there are any interactions present
 - [ ] My issue disappears when I hold shift while starting Anki.
 - [x] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 

#### What Anki version are you using?

*Please open Anki, go to Help → About (windows) or Anki → About Anki (mac) to find the version. Edit the value between the backticks below

```
24.06.3 (d678e393)⁩
```

#### Error message (if any)

*If you've received an error message, please copy and paste it between the backticks below:*

```

```

**Comment by AnKingMed** [comment]
Is it working on desktop on that card? Just not on mobile?  

Did you sync from desktop to mobile after enabling it?

**Comment by marchjune2027** [comment]
I just got the addon with the latest anki, and it's not working for me either. I definitely know theres 5+ cards for gout, but it says no results when i hover my mouse over it

**Comment by AnKingMed** [comment]
@DoctorToBeIn23 @abdnh can you look into this?

**Comment by abdnh** [comment]
@marchjune2027 Can you attach a sample deck?

**Comment by marchjune2027** [comment]
I have the step 3 and sketchy pharm deck. Do you want me to attach those on here?

**Comment by DoctorToBeIn23** [comment]
Have you tried to just touch the word, you dont need to highlight it...

**Comment by marchjune2027** [comment]
yes, when i hover over it, it says 'no results' but it's only showing this for one of my decks. For the other decks, it doesn't even show 'no results'. I've attached a photo of my settings.

<img width="889" alt="Image" src="https://github.com/user-attachments/assets/4da5dcd4-8823-464d-a687-529b77f42cc1" />
<img width="896" alt="Image" src="https://github.com/user-attachments/assets/5872ca8e-5e52-49e4-8b0e-d1665ad215cc" />

**Comment by marchjune2027** [comment]
so for my personal deck, when i move my mouse on the word, this is what it shows

<img width="1424" alt="Image" src="https://github.com/user-attachments/assets/9173b252-7f3a-47a3-9621-46487afd369e" />

**Comment by marchjune2027** [comment]
but for the step 3 deck i downloaded from reddit, when my mouse is on the word, nothing pops up, not even 'no results'

**Comment by marchjune2027** [comment]
I actually unchecked all from the 'exclude field from search' and it's now working for my personal deck! thank you!!
but my Step 3 deck, still nothing pops up

**Comment by abdnh** [comment]
> but my Step 3 deck, still nothing pops up

If you're sure the note types used in this deck are enabled, please attach a sample of the deck (a couple of cards will do).

**Comment by marchjune2027** [comment]
I tried to attach just 6 card deck but it says file type not allowed .apkg

**Comment by marchjune2027** [comment]
<img width="557" alt="Image" src="https://github.com/user-attachments/assets/da51b61a-f846-477b-b20f-abe1073f3c5e" />
<img width="579" alt="Image" src="https://github.com/user-attachments/assets/ecbb5e2a-25c7-43ce-88d0-6708df9922df" />
<img width="988" alt="Image" src="https://github.com/user-attachments/assets/5ad1f886-f0d0-4ca8-b9cd-8bb60ce70f97" />
<img width="1051" alt="Image" src="https://github.com/user-attachments/assets/085f21d4-1f1b-4f60-a8ad-d9f62c881a82" />
<img width="1163" alt="Image" src="https://github.com/user-attachments/assets/154a199b-30f6-4c26-a5ef-1c858ce8b948" />

**Comment by AnKingMed** [comment]
You can just zip them and attach it

**Comment by marchjune2027** [comment]
[Step 3__RheumAlk.apkg.zip](https://github.com/user-attachments/files/18629041/Step.3__RheumAlk.apkg.zip)

**Comment by marchjune2027** [comment]
got it! I think it worked?

**Comment by abdnh** [comment]
I can't reproduce the issue.

The notes in your deck use the "J" note type. Can you confirm it's enabled in the add-on settings?

![Image](https://github.com/user-attachments/assets/a9d16ea6-ca93-4d54-ae10-657ab31478e0)

**Comment by marchjune2027** [comment]
I clicked toggle all so that it selects all again, but it's giving me this error

<img width="1145" alt="Image" src="https://github.com/user-attachments/assets/93a93f50-e8fe-42d1-a450-092b150e4d2b" />

**Comment by abdnh** [comment]
> I clicked toggle all so that it selects all again, but it's giving me this error

You have a note type with a template error, which is preventing the add-on from processing it. Try excluding this type (Cloze-c2626) or deleting it from _Tools > Manage Notetypes_ if it's not used by any notes.

**Comment by marchjune2027** [comment]
That worked! thank you SO much for all your help!!
